iris/admin/__init__.py

```diff
@@ -55,13 +56,20 @@ def actions(type):
     order_by = flask.request.args.get('order_by', 'user_id')
     ascending = flask.request.args.get('ascending', 'true')
     ascending = True if ascending == 'true' else False
+    combine_masks = flask.request.args.get('combine_masks', 'false')
+    combine_masks = True if combine_masks == 'true' else False
 
     actions = Action.query.filter_by(type=type)
     if ascending:
         actions = actions.order_by(getattr(Action, order_by)).all()
     else:
         actions = actions.order_by(getattr(Action, order_by).desc()).all()
 
+    if combine_masks:
+        for i in set(action.image_id for action in actions):
```

Comment:
I think this could do with a comment here. If I've understood correctly it'd be something along the lines of "call merge_masks for each unique image_id". I'd also rename the var i to something more descriptive like `image_id` since `i` to me implies a for loop counter.